pygmt/src/ternary.py

```diff
@@ -0,0 +1,59 @@
+"""
+Histogram - Create a histogram
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    B="frame",
+    C="cmap",
+    F="center",
+    G="fill",
+    J="projection",
```

Comment:
Seems like `tenary` uses [`-JX`](https://docs.generic-mapping-tools.org/6.3/ternary.html#j) only, so should we alias it like so?

```suggestion
    JX="projection",
```